index.html

```diff
@@ -24,29 +25,33 @@
         </section>
         <nav class="navbar navbar-expand-lg mt-0">
```

Comment:
- Optional
It would be better if the mobile navbar would close when the user clicks on any link which takes the user to a section on the same page. It would look much better for the user experience. Right now, the navbar stays open all the time and only closes when i click the close button.